site/docs/guidelines/forms.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Forms
+navTitle: Forms
+summaryParagraph: Use forms to allow people to enter data for use by the application, or to configure options. Designing forms well requires making decisions about structure, sequence, interface elements, field labels, help, and feedback. 
+  Alignment creates visual connections, a sense of order, and hierarchy. Horizontal rhythm keeps horizontal spaces between elements on a page consistent.
+tags: ['Inputs','Labels', 'Placeholders', 'Validation', 'Form layout']
+needToKnow:
+  - "Every form has a primary and secondary action."
+---
+
+## To keep in mind
+
+### Layout
+
+See individual component pages for label and input layout.
+We optimize for readability, scannability and time to form completion.
+
+Our preferred layout is with individual form elements stacked vertically, grouped by context as necessary. 
+
+
+#### Actions
+
+We require a single primary action to submit, with a secondary action to cancel and discard edits.
+
+### Accessibility
+
+All form elements must have a label associated with the input. There are reasons you might make it visible to sighted users or not, see individual component pages for details.
+
+### Validation and feedback
+
+Where possible use client side validation where the user is presented feedback before submitting the form.
+
+
+## See also
+- [Link vs button](/guidelines/link-vs-button)
+- [Button](/components/button)
+- [Checkbox Field](/components/checkbox-field)
+- [Checkbox Group](/components/checkbox-group)
+- [Date Picker](/components/date-picker)
+- [Icon Button](/components/icon-button)
+- [Icon](/components/icon)
+- [Radio Group](/components/radio-group)
+- [Text Field](/components/text-field)
+- [Autocomplete](/components/autocomplete)
+- [Divider](/components/divider)
+- [Dropdown](/components/dropdown)
+- [Divider](/components/divider)
```

Comment:
We have a duplicate of Divider in this list. Let's delete that with the final check of other pages to link.